.circleci/config.yml

```diff
@@ -276,6 +276,8 @@ jobs:
       - *setsvsdispname
       - *setmetapackageversion
       - *builddebianpackage
+      - *addsshkeys
+      - *commitworkstationdebs
```

Comment:
heads up this job runs per-PR

per package generally we have two CI jobs, one running per-PR that builds the latest released version (see: `build-buster-securedrop-export`), and one running nightly (see: `build-nightly-buster-securedrop-export`) that bumps the version and after build commits the package

for this change we should add a new nightly job and leave this `build-buster-securedrop-workstation-svs-disp` as is

Reply:
addressed, good call